WooCommerce/Classes/POS/Presentation/CardReaderConnection/UI States/Reader Messages/PointOfSaleCardPresentPaymentDisconnectedMessageView.swift

```diff
@@ -1,7 +1,8 @@
 import SwiftUI
 
 struct PointOfSaleCardPresentPaymentReaderDisconnectedMessageView: View {
-    let viewModel: PointOfSaleCardPresentPaymentReaderDisconnectedMessageViewModel
+    private let viewModel = PointOfSaleCardPresentPaymentReaderDisconnectedMessageViewModel()
```

Comment:
> Not having to do any manual injection here really simplifies the view and view model, in my opinion.

Yesss. Although somehow still feels so wrong 😀 Need to unlearn the ways of `UIKit`